deploy-board/deploy_board/webapp/helpers/accounts_helper.py

```diff
@@ -27,3 +28,17 @@ def get_by_cell_and_id(request, cell, account_id, provider="AWS"):
 
 def get_default_account(request, cell, provider="AWS"):
     return get_by_cell_and_id(request, cell, "default", provider)
+
+def get_aws_owner_id_for_cluster_name(request, cluster_name):
+    cluster = rodimus_client.get("/clusters/%s" % cluster_name, request.teletraan_user_id.token)
```

Comment:
Could be nice to reuse the [`clusters_helper.get_cluster`](https://github.com/pinterest/teletraan/blob/master/deploy-board/deploy_board/webapp/helpers/clusters_helper.py#L52) call to keep the call in one place

Reply:
Updated, thanks.